### PR TITLE
Enable multiplex and unified certs by default

### PR DESCRIPTION
**Pull request by situ-s**
* **Problem Overview**  
  Enable multiplex and unified_certificates by default
  Apollo support to enable unified certificates per test case  
* **Testing Done** 
 Apollo CI passed locally https://github.com/situ-s/concord-bft/actions/runs/2740259301
 Concord CI testing in progress
 System testing is done by enabling lower end workloads

### CODE REVIEW

**Comment by teoparvanov** [comment]
Hi @situ-s and @yuliasherman , does this mean TLS multiplexing will be enabled by default for both DAML and Ethereum?

Thanks for the clarification,
Teo

File: client/bftclient/include/bftclient/config.h [hunk 1]
```diff
@@ -66,6 +66,7 @@ struct ClientConfig {
   uint16_t f_val;
   uint16_t c_val;
   RetryTimeoutConfig retry_timeout_config;
+  bool unified_certs = false;
```
Comment (yuliasherman) [inline client/bftclient/include/bftclient/config.h:69]: Please replace unified_certs by use_unified_certs everywhere below to have a consistent parameter name.

**Review by yuliasherman — APPROVED** [review]
LGTM

File: client/bftclient/include/bftclient/config.h (hunk 1 above)
Comment (situ-s) [reply client/bftclient/include/bftclient/config.h]: made the changes

**Comment by teoparvanov** [comment]
> > Hi @situ-s and @yuliasherman , does this mean TLS multiplexing will be enabled by default for both DAML and Ethereum?
> > Thanks for the clarification, Teo
> 
> Only for DAML

Isn't Ethereum higher priority at the moment?

**Comment by eyalrund** [comment]
we start with DAML (already implemented and tested). Ethereum is next

**Review by eyalrund — APPROVED** [review]

**Pull request merged**